retrofit/src/main/java/retrofit2/Call.java

```diff
@@ -70,4 +70,7 @@
 
   /** The original HTTP request. */
   Request request();
+
+  /** The service method invocation that created this call. */
+  Invocation invocation();
```

Comment:
I can't think of a reason to expose this here. In my version I only had it added as a request tag. You can still access it from here by calling `request().tag(Invocation.class)` although I expect no one to do that.

Reply:
SGTM!